instrumentserver/dashboard/config.py

```diff
@@ -0,0 +1,56 @@
+config = {
```

Comment:
this file looks like an example, right? 
maybe it'd be better make this more clear?
My favorite option would be to have an 'example' folder in the repository root directory.